apps/chat-e2e/src/assertions/folderAssertion.ts

```diff
@@ -418,4 +424,51 @@ export class FolderAssertion<T extends Folders> {
       .soft(actualFoldersCount, ExpectedMessages.foldersCountIsValid)
       .toBe(expectedCount);
   }
+
+  public async assertFoldersSorting(sorting: Sorting) {
```

Comment:
I think it would not work with the nested folder structure. And you also do not pass the folder/folders to the method that is not consistent with all the other class methods. 

As a general soultiuon, I think it would be much more convenient if you create a generic method assertNamesSorting that can assert the sorting af any entities or any array of strings?

Reply:
moved to a generic method